src/enumeration.rs

```diff
@@ -101,6 +104,12 @@ impl DeviceInfo {
         self.driver.as_deref()
     }
 
+    /// *(Windows-only)* Location paths of this device
+    #[cfg(target_os = "windows")]
+    pub fn location_paths(&self) -> Vec<&OsStr> {
```

Comment:
`Vec<&OsStr>` seems like an odd type mixing the owned Vec and borrowed OsStr. Why not `&[OsString]` or `impl Iterator<Item = &OsStr>`, which can be returned without reallocating?

Reply:
The return type has been fixed.